lntest/bitcoind_common.go

```diff
@@ -64,6 +64,11 @@ func (b BitcoindBackendConfig) DisconnectMiner() error {
 	return b.rpcClient.AddNode(b.minerAddr, rpcclient.ANRemove)
 }
 
+// Credentials returns the rpc username, password and host for the backend.
+func (b BitcoindBackendConfig) Credentials() (string, string, string, error) {
```

Comment:
You need the same method in the `BtcdBackendConfig` as well. That's why the linter and all tests currently fail.